### PR TITLE
Removing sections

### PR DESCRIPTION
**Pull request by ZELFs**
Removing sections 
due to: 
4(.2) Unnecessary, already integrated part of the vouching for, I believe
5(.2) I don't think we should make a practice of assuming that people can contribute with unpaid work. Also, I don't think we should make it a practice to count and keep tabs on each other.

### CODE REVIEW

**Comment by arj03** [comment]
I feel that the section 4.2 should stay. Without this there isn't really a clear definition of what we look for in people that wants to join. It comes back to if we want this to be super inclusive or a tighter group. The trust principle suggest that we aim for a tighter group.

**Comment by cblgh** [comment]
@ZELFs we decided to postpone any decision on this since you were missing, see the discussion here: https://workflowy.com/s/buttsortium/vgLcKEJhkfeVPxp8#/879dfc1162af

**Comment by cblgh** [comment]
@ZELFs we decided to postpone any decision on this since you were missing, see the discussion in the workflowy document under today's meeting 2020-04-08

**Comment by Powersource** [comment]
We voted yes on "Proposal: Remove 4.2 and adds a section in the "Reasoning" document explaining what we are looking for in new members (ex: contributions to SSB)"

**Comment by Powersource** [comment]
Suggestion: Change 5.2 from

> Performed >=10h of recognized volunteering work for the consortium over the last 12 months, beside voting and participation in meetings.

to

> Performed volunteering work for the consortium over the last 12 months, beside voting and participation in meetings.

**Comment by arj03** [comment]
I have reflected on section 5.2 and for me the are two aspects that I liked about the original proposal:
 - the 10h signals a commitment shared by all members, I think its important that we establish what we expect of other people. For me the 10h specifically is not so important, I think it's more important to have a process.
 - What I like about *recognized* is that it brings about transparency. I can see how it, combined with hourly tracking, can be seen as *keep tabs on each other*. So if we instead can have a process where progress is shared (like there is a good habit on SSB of doing), then I'm fine with removing it. Right now it is not a problem because we are so few, but if we organization scales and we start working in smaller groups, then this becomes very important I think.

So in summary if we adopt changes that address these points, could be in the reasoning document, then I'm good with the revised version Powersource posted.

**Comment by ZELFs** [comment]
I fully understand that there is a need for establishing shared expectations, and I agree that this is very important. Not only in the work of the buttsortium but in work in general, as me and Eric learned a lot about during the Handshake Council. 

I also want to have ways to establish expectations. I have a suggestion: based on the previous agreement of the formation being working groups (pods, cirlcles, however we call them) I believe that the choice as to *how* those expectations are established can be taken in the groups people work in, which is also where the time is actively put in. A proposal for how this could work has been made [here](https://github.com/scuttlebutt-eu/important-documents/pull/34/files).   

The reasons I believe this is a good approach is for three main reasons:

1) Keeping to the philosophy of letting the base framework be minimal for a start to enable it to evolve with time. The reasons for this is to not over design at this stage, as it might hinder our process in the future if we have to backtrack, and to ensure that the guidelines of the consortium are built by the members, rather then being imposed upon members by the initiators. 

2) Structure is Communication. Having a set amount of minimum volunteering hours communicates things. Two aspects I see communicated through this, beyond the prior, is 1, The Buttsortium has a value set which assumes that people participating can afford volunteering. 2, The hours are set which opens up for the possibility of imposing counting and tracking of each other, a surveillance based relationship between members. Both of these I feel highly uncomfortable with. 

3) Following the Energy. Leaving space for peoples capacities to vary over time. If one initiative slows down or the people no longer are capable of contributing, this is a natural energy decent, and informs what is happening. I believe it's important to listen to these indicators. One way to listen to this is to have regularly scheduled meta-conversations about the process.

**Comment by elavoie** [comment]
@ZELFs 

> Keeping to the philosophy of letting the base framework be minimal for a start to enable it to evolve with time. 

 Establishing a minimum number of hours of contribution does not imply complicated processes. We can say that members could simply mention what they contributed on in the last year in a post in their preferred platform and link to that for their membership renewal. We could simply trust them that it represented at least 10h of work and be done with it.

Compared to that, the current proposal requires the formation of a working group, presumably with someone taking leadership on doing it, and the report/non-report of the group on the commitment of members. That is to say, it represents no commitment at all.

> The reasons for this is to not over design at this stage, as it might hinder our process in the future if we have to backtrack,

Backtracking has very low cost. If we end up with people that we would really like to have in the consortium but do not qualify because they cannot commit 10h/year, and that is critical to the workings of the consortium, we simply vote that rule out.

>  and to ensure that the guidelines of the consortium are built by the members, rather then being imposed upon members by the initiators.

We are yet back to abstract ideas of *coercion*. The same argument could be made with regard to choosing a name for the consortium, or any other rules or guidelines we are making. At some point, we need to make a number of decisions to get something up and running. I would argue, as in my essay on Commitment, that unless members of an organization collectively commit at least enough resources in time and/or money to at least maintain/grow the organization, the organization will not survive in the long run because the responsibilities and cost of maintaining the organization will fall on too little people that will eventually burn out.

The *power* to shape an organization future, through democratic decision making, and the *benefits* derived from participating in a group initiative, must be equally balanced by a *responsibility* in keeping the group working.

> Structure is Communication. Having a set amount of minimum volunteering hours communicates things. Two aspects I see communicated through this, beyond the prior, is 1, The Buttsortium has a value set which assumes that people participating can afford volunteering. 

Let me turn that around. Suppose some members want to participate and vote on decisions of the consortium and have *no money* to contribute, and *no time* to participate (not even *1h/month* or *10h/year*). What exactly is their contribution in the consortium? And why should others want to give them decision power over what they themselves are contributing? 

> 2, The hours are set which opens up for the possibility of imposing counting and tracking of each other, a surveillance based relationship between members. Both of these I feel highly uncomfortable with.

I don't understand why that comes back again, and I would like to understand better what exactly makes you uncomfortable. I had thought I had successfully given one example, in my Commitment essay, of how this could be done without coercion or surveillance. Here is the summary:

> > A commitment to me is a *willfull* use of *personal freedom* towards *self-chosen and publicly shared aims*. Commitment increases the reach of what a community or organization can do: it enables members to better focus on their tasks and issues by trusting that other tasks and responsibilities will be taken care of by others. There is always a risk that this trust may be broken, but with proper personal responsibility in dealing with events outside of ones’ control and with the internal practice of focus and discipline, this trust can be maintained without outside coercive controls.

> Following the Energy. Leaving space for peoples capacities to vary over time. If one initiative slows down or the people no longer are capable of contributing, this is a natural energy decent, and informs what is happening. I believe it's important to listen to these indicators. One way to listen to this is to have regularly scheduled meta-conversations about the process.

I agree this is important but I don't see how this applies to membership criteria. 

@ZELFs It seems to me that you are against *any level* of commitment/responsibility whatsoever *in principle*, while I am trying to establish a *minimal level of commitment*. These two things are incompatible.

From my personal experience in groupwork, with roommates, with the Handshake Council, and observing the dynamics of SSB over the last years, the best groupwork I have participated, and the most fun I have had, had always been when I could *rely* on my teammates and collaborators, so I could focus on one thing and know the others would be done. I have had repeated frustrations around *unclear* responsibilities and commitment levels, *unreliable* presence and *unpredictable* participation, and in repeated cases I have taken up the extra slack because I cared that things still got done. I don't see how any of the suggestions made so far address this.

**Comment by elavoie** [comment]
I would also add that there is nothing new here and the sharing of responsibility and commitments is a common pain point in organizing intentional communities. See [Diana Leaf Christian](https://en.wikipedia.org/wiki/Diana_Leafe_Christian), [Creating a Life Together: Practical Tools to Grow Ecovillages and Intentional Communities](https://dianaleafechristian.org/creating_a_life_together_practical_tools_to_grow_ecovillages_and_intentional_communities.html).

**Comment by Powersource** [comment]
5.2 (previously/in this PR called 4.2) was voted on and will be handled by https://github.com/scuttlebutt-eu/important-documents/issues/32

6.2 (previously/in this PR called 5.2) was resolved with https://github.com/scuttlebutt-eu/important-documents/pull/37

So my interpretation is that this PR is superfluous/has moved forward in different forms, so I'll close this now. Reopen if someone has a different interpretation.
